ops/ansible/ansible_upgrade.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+pip install ansible
```

Comment:
I overlooked that. I have the `--upgrade ansible setuptools` in the install.sh script because ansible throws errors if you don't pip install setuptools after you install it from apt. It's weird I know. 
And yeah we want ansible to be 2.0+ for sure.